third_party/intel/lib/TritonAnnotateModule/TritonAnnotateModule.cpp

```diff
@@ -0,0 +1,69 @@
+#include "intel/include/Analysis/DPAS.h"
+#include "intel/include/TritonAnnotateModule/Passes.h"
+#include "triton/Dialect/TritonGPU/IR/Dialect.h"
+
+namespace mlir::triton::gpu::intel {
+#define GEN_PASS_DEF_TRITONANNOTATEMODULE
+#include "intel/include/TritonAnnotateModule/Passes.h.inc"
+} // namespace mlir::triton::gpu::intel
+
+using namespace mlir;
+using namespace mlir::triton::gpu;
+
+namespace {
+
+constexpr char AttrLTS[] = "triton_gpu.is_lts";
+constexpr char AttrTargetName[] = "triton_gpu.target";
+
+struct TritonAnnotateModule
+    : public intel::impl::TritonAnnotateModuleBase<TritonAnnotateModule> {
+  using Base::Base;
+
+  void runOnOperation() override {
+    ModuleOp mod = getOperation();
```

Comment:
```suggestion
    ModuleOp mod = getOperation();
    Builder builder(mod);
```

Create builder to use below